src/compositemark/boxplot.ts

```diff
@@ -1,4 +1,4 @@
-import {isNumber} from 'vega-util';
+import {isNumber} from 'util';
```

Comment:
`util.isNumber` is deprecated. If `vega-util` has `isNumber` function, Is there is a reason why you are using it from `util`?

Reply:
Good catch! #3446